CONTRIBUTING.md

```diff
@@ -0,0 +1,248 @@
+# Contributing
+
+Thank you for wanting to contribute to *aws-nuke*.
+
+Because of the amount of AWS services and their rate of change, we rely on your
+participation. For the same reason we can only act retroactive on changes of
+AWS services. Otherwise it would be a fulltime job to keep up with AWS.
+
+
+## How Can I Contribute?
+
+### Some Resource Is Not Supported by *aws-nuke*
+
+If a resource is not supported yet by *aws-nuke*, you have two options to
+resolve this:
+
+* File [an issue](https://github.com/rebuy-de/aws-nuke/issues/new) and describe
+  what resource is missing. We will take care and add it as soon as we have
+  some time.
```

Comment:
Looking back at last year, we probably won't? I'd rather write "This way we or others can take care of it..."